tests/benchmarks/http-benchmarks/jvm/src/aws/smithy/kotlin/benchmarks/http/HttpEngineBenchmarks.kt

```diff
@@ -54,7 +57,7 @@ private val largeData = ByteArray(MB_PER_THROUGHPUT_OP * 1024 * 1024)
 @State(Scope.Benchmark)
 @OutputTimeUnit(TimeUnit.SECONDS)
 open class HttpEngineBenchmarks {
-    @Param(OKHTTP_ENGINE, CRT_ENGINE)
+    @Param(OKHTTP_ENGINE, CRT_ENGINE, OKHTTP4_ENGINE)
     var httpClientName: String = ""
```

Comment:
**Correctness**: Thanks for adding the new engine to the benchmark suite. Can you please run the benchmarks and update [the **README**](https://github.com/smithy-lang/smithy-kotlin/blob/84a601373ce9584541b885feef1eaa5434e0f04a/tests/benchmarks/http-benchmarks/README.md) with the results?

Feel free to use whatever instance type and JDK are convenient.